Open62541.xs

```diff
@@ -2130,11 +2130,14 @@ UA_Client_sendAsyncBrowseRequest(client, request, callback, data, reqId)
 	SV *				callback
 	SV *				data
 	OPCUA_Open62541_UInt32		reqId
+    INIT:
+	if (SvOK(ST(4)) && !(SvROK(ST(4)) && SvTYPE(SvRV(ST(4))) < SVt_PVAV))
```

Comment:
I'm guessing this means no more passing undef for reqId.
Do we need this restriction here? Unlike the other functions the "real" response is retrieved in the callback.

Reply:
My mistake - undef is still allowed in this case